src/components/Moteoversikt.tsx

```diff
@@ -46,14 +45,14 @@ const Moteoversikt = (): ReactElement => {
 
   return (
     <>
-      <Verktoylinje>
+      <div className="flex mb-8">
         <MoteResponsFilter
           moteResponser={getMoteResponser(moter)}
           onFilterChange={(changedFilter: MoteRespons) =>
             setResponsFilter(changedFilter)
           }
         />
-      </Verktoylinje>
+      </div>
```

Comment:
Ikke kjempeviktig. Bare lurte på om styling kunne ligge direkte på `MoteResponsFilter`? 😊

Reply:
`MoteResponsFilter` brukes to steder så blir litt mer komplisert å legge stylingen der.

Reply:
Skjønner! Nevermind da 😊